notebooks/concepts/Specviz from splot concept notebook.ipynb

```diff
@@ -0,0 +1,3636 @@
+   "metadata": {},
+   "source": [
+    "## Uncertainties/mask"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If the `Spectrum1D` object loaded contains an uncertainty and/or mask attribute, these should be loaded ointo glue when `load_data` is called.  However, by default only the spectrum itself should be *shown* at that time. The user can show it later using standard `jdaviz` interactions if desired. We *may* wish some syntactic convenience for the notebook as well - e.g.:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "specviz.show_uncertainty()\n",
+    "specviz.show_mask()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Accessing the spectrum\n",
+    "\n",
+    "(only relevant for notebook/developer usage) - When a single spectrum is loaded using `load_data`, it should be easy to get that spectrum back again.  E.g.,"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "spec = specviz.current_spectrum  #<- a Spectrum1D object\n",
+    "spec.flux, spec.uncertainty, ..."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "However, this is ambiguous if multiple spectra are loaded (see the `add_data` discussion above). Addressing this ambiguity would be possible using the glue-level names associated with these data:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "spec = specviz.app.get_viewer_data('spectrum-with-some-name')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A middle ground might be for there to be a special case of \"whichever spectrum was last loaded via `load_data`', which is special state that the specviz helper remembers.  In that case the helper needs a version of `get_viewer_data` which only differs in how it behaves in the no-argument case:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "spec = specviz.get_viewer_data()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Which would be the same as the `current_spectrum` option above.\n",
+    "\n",
+    "In the below, `specviz.current_spectrum` is used for simplicity, but the intent is that that would instead be `get_viewer_data` if the above suggestion is followed."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## SPLOT commands addressed by data loading operations "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Options: images\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "The actual spectrum file that gets loaded when splot starts"
+   ]
```

Comment:
What is meant by images? Do we want the option to inspect a 2D (dispersed image) of a spectrum, or will this be strictly a 1D spectrum tool?

Reply:
That's a holdover from SPLOT, which usage the term "image" generically for all fits file regardless of their dimensionality.  I'll try to clarify that in the text here, though.